alert-common/src/main/java/com/synopsys/integration/alert/common/message/model/ComponentKey.java

```diff
@@ -97,4 +100,11 @@ public boolean equals(final Object otherObject) {
         return false;
     }
 
+    @Override
+    public int compareTo(final ComponentKey other) {
```

Comment:
Should either `compareTo` or `equals` reference the other so that we do not have duplication?